providers/google/semantic_location_history_test.go

```diff
@@ -0,0 +1,154 @@
+	placeVisits[0].ChildVisits[0].OtherCandidateLocations[0].PlaceVisitID = 0
+	assert.EqualValues(t, expectedPlaceVisit, placeVisits[0])
+
+	expectedActivitySegment := ActivitySegment{
+		Activities: []ActivityTypeCandidate{
+			{ActivityType: "IN_PASSENGER_VEHICLE", Probability: 93.55993270874023},
+		},
+		ActivityType:                       "IN_PASSENGER_VEHICLE",
+		Confidence:                         "HIGH",
+		Distance:                           13958,
+		DurationEndTimestampMs:             types.DateTime(time.Date(2020, 2, 1, 20, 1, 43, 307000000, time.UTC)),
+		DurationStartTimestampMs:           types.DateTime(time.Date(2020, 2, 1, 19, 22, 14, 623000000, time.UTC)),
+		EndLocationLatitudeE7:              340322573,
+		EndLocationLongitudeE7:             -1184817511,
+		ParkingEventLocationAccuracyMetres: 125,
+		ParkingEventLocationLatitudeE7:     340320032,
+		ParkingEventLocationLongitudeE7:    -1184827155,
+		ParkingEventTimestampMs:            types.DateTime(time.Date(2020, 2, 1, 20, 4, 20, 47000000, time.UTC)),
+		SimplifiedRawPathPoints: []ActivityPathPoint{
+			{
+				AccuracyMeters: 65, LatE7: 340779839, LngE7: -1184296112,
+				Time: types.DateTime(time.Date(2020, 2, 1, 19, 26, 45, 571000000, time.UTC)),
+			},
+		},
+		StartLocationLatitudeE7:  340902329,
+		StartLocationLongitudeE7: -1184383023,
+		TransitPathHexRgbColor:   "ED7D31",
+		TransitPathName:          "33",
+		TransitStops: []TransitStop{
+			{
+				LatitudeE7: 339888109, LongitudeE7: -1184626624,
+				PlaceID: "ChIJZYma8pW6woARV34ul5SMRSk",
+				Name:    "Venice / Abbot Kinney",
+			},
+		},
+		Waypoints: []Waypoint{
+			{LatE7: 340902481, LngE7: -1184381866},
+		},
+	}
+	activitySegments[0].Model = gorm.Model{}
+	activitySegments[0].Activities[0].Model = gorm.Model{}
+	activitySegments[0].Activities[0].ActivitySegmentID = 0
+	activitySegments[0].Activities[0].ActivitySegment = ActivitySegment{}
+	activitySegments[0].SimplifiedRawPathPoints[0].Model = gorm.Model{}
+	activitySegments[0].SimplifiedRawPathPoints[0].ActivitySegmentID = 0
+	activitySegments[0].SimplifiedRawPathPoints[0].ActivitySegment = ActivitySegment{}
+	activitySegments[0].TransitStops[0].Model = gorm.Model{}
+	activitySegments[0].TransitStops[0].ActivitySegmentID = 0
+	activitySegments[0].TransitStops[0].ActivitySegment = ActivitySegment{}
+	activitySegments[0].Waypoints[0].Model = gorm.Model{}
+	activitySegments[0].Waypoints[0].ActivitySegmentID = 0
+	activitySegments[0].Waypoints[0].ActivitySegment = ActivitySegment{}
+	assert.EqualValues(t, expectedActivitySegment, activitySegments[0])
+
+	placeVisits = nil
+	activitySegments = nil
+	require.NoError(t, p.importSemanticLocationHistoryFromDirectory("testdata/google/Location History/Semantic Location History/"))
+	require.NoError(t, db.Find(&placeVisits).Error)
+	require.NoError(t, db.Find(&activitySegments).Error)
+	require.Len(t, placeVisits, 2)
+	require.Len(t, activitySegments, 1)
```

Comment:
how's about also comparing before and after import like here?
https://github.com/bionic-dev/bionic/blob/55d68c6833519652a94db309fdf236f20fe80e41/providers/rescuetime/activity_history_test.go#L50